coursebuilder/modules/explorer/_static/components/pages/certification.html

```diff
@@ -30,61 +38,68 @@ <h1 class="intro-title">Get Certified by Google</h1>
       <!--  Awards  -->
       <div class="section awards">
         <div class="carousel-section-wrapper">
-          <h2 class="section-label">An award winning platform</h2>
+          <h2 class="section-label"></h2>
           <div class="owl-carousel owl-theme awards-carousel">
+            <!-- <div class="item"> -->
             <div class="item">
-              <img src="/modules/explorer/_static/images/award2.png" alt="">
-              <p>
-                UK Digital Experience Award <br> Customer Experience Innovation
-              </p>
-            </div>
-            <div class="item">
-              <img src="/modules/explorer/_static/images/digiday_award.png" alt="">
-              <p>
-                Digiday Awards Europe <br>
-                Best Audience Development Platform
-              </p>
-            </div>
-            <div class="item">
-              <img src="/modules/explorer/_static/images/digiday_award.png" alt="">
-              <p>
-                Digiday Awards Europe <br>
-                B2B Campaign of the Year
-              </p>
-            </div>
-            <div class="item">
-              <img src="/modules/explorer/_static/images/award3.png" alt="">
-              <p>
-                Czech Internet Award <br> Market Inspiration 2016 - Silver Award
-              </p>
-            </div>
-            <div class="item">
-              <img src="/modules/explorer/_static/images/europe_award.png" alt="">
-              <p>
-                European Commission <br> Digital Skills Award 2016
-              </p>
-            </div>
-            <div class="item">
-              <img src="/modules/explorer/_static/images/award1.png" alt="">
-              <p>
-                Marcom Platinum Award <br> Economic Development
-              </p>
-            </div>
-            <div class="item">
-              <img src="/modules/explorer/_static/images/award1.png" alt="">
-              <p>
-                Marcom Gold Award <br> App for Training/Learning
-              </p>
-            </div>
-            <div class="item">
-              <img src="/modules/explorer/_static/images/award3.png" alt="">
-              <p>
-                European Commission <br> Digital Skills Award 2016
-              </p>
+                <a href="https://www.facebook.com/HacksHackersAfrica">
+                  <img src="/modules/explorer/_static/images/hha.gif" alt="Hack Hackers Africa">
+                  <p>
+                </a>    
```

Comment:
@andela-iakande Close in the right place.

Reply:
fixed